.travis.yml

```diff
@@ -72,6 +73,22 @@ matrix:
     - env: BUILD_TYPE="Debug" WITH_SYMENGINE_ASSERT="yes" WITH_PYTHON="yes" PYTHON_VERSION="2.7"
       compiler: gcc
       os: osx
+    - language: ruby
+      cache: bundler
+      env: TEST_IN_TREE="yes" WITH_RUBY="yes"
+      rvm: 1.9.3
```

Comment:
It seems travis is not using the compiler variable when the language is set to ruby.
So compiler for each ruby build should be `gcc`